scripts/transpilation/copy-css.js

```diff
@@ -30,8 +30,7 @@ const cssFiles = execSync('find packages -name "*.css" | grep -v node_modules')
 cssFiles.map((cssFile) => {
   let component;
   const paths = cssFile.split('packages/');
-  const regEx = new RegExp('/');
-  const componentPath = paths[1].split(regEx);
+  const componentPath = paths[1].split('/');
```

Comment:
ESLint was complaining about using the RegExp constructor instead of using a regex directly

Reply:
Guessing this was also checked post this change and things are still happy?

Reply:
I haven't tested it actually 🤔 I thought they're exactly the same thing, but I'll check it!